app/helpers/podcasts_helper.rb

```diff
@@ -75,4 +81,8 @@ def podcast_retry_image_path(podcast, feed_form, image_form)
       edit_podcast_path(podcast, uploads_retry_params(feed_form, image_form))
     end
   end
+
+  def rss_language_options
```

Comment:
So I'd say just leave `NON_COMPLIANT_CODES` out of the dropdown entirely.  Avoid any confusion from the user as to why they're there.

And if you pass the podcast into this method, you can check:

```ruby
if RSS_LANGUAGE_CODES.include?(podcast.language)
  # return just the language_codes
else
  # return the language_codes.concat podcast.language
end
```